marklogic-data-hub/src/main/java/com/marklogic/hub/flow/ProvenanceManager.java

```diff
@@ -0,0 +1,63 @@
+package com.marklogic.hub.flow;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.dataservices.ExecCaller;
+import com.marklogic.client.dataservices.IOEndpoint;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.hub.HubClient;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ProvenanceManager {
```

Comment:
Almost all of this code is the same as in JobManager - the only difference appears to be the API path. So we can make a static utility method like this:

```
public static void deleteData(String apiPath, String retainDuration)
```

Because that class will be specific to classes in the com.marklogic.client.dataservices package, and specifically the "bulk" classes, let's put that method into a new com.marklogic.hub.dataservices.BulkUtil class. Both JobManager and ProvenanceManager can use it and avoid all of this duplicated code.